src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common.Tooling/LanguageServer/CSharpTestLspServerHelpers.cs

```diff
@@ -179,8 +179,7 @@ private class EmptyMappingService : IRazorSpanMappingService
     {
         public Task<ImmutableArray<RazorMappedSpanResult>> MapSpansAsync(Document document, IEnumerable<TextSpan> spans, CancellationToken cancellationToken)
         {
-            var result = Enumerable.Empty<RazorMappedSpanResult>().ToImmutableArray();
```

Comment:
Yeah, I was going to leave some of the `ImmutableArray` tweaks out of this PR because they're off-topic. However, this was the line that was so egregious, that I kept those changes in. :smile: